docs/developer-docs/latest/setup-deployment-guides/installation/templates.md

```diff
@@ -133,9 +155,7 @@ After reading the above rules, follow these steps to create your template:
 
 1. Create a standard Strapi app with `create-strapi-app`, using the `--quickstart` option.
 2. Customize your app to match the needs of your use case.
-3. Outside of Strapi, create a new directory for your template.
-4. Create a `template.json` file in your template directory.
+3. Generate your template by running `strapi generate:template <path>`
```

Comment:
I would add a link to the the corresponding section in the CLI guide here
(the other file you updated in this PR)

Thank you!